.travis.yml

```diff
@@ -16,6 +17,7 @@ matrix:
         - py.test --pep8 -m pep8
 
 before_install:
+  - sudo apt-get -y -q install ffmpeg libavcodec-extra
```

Comment:
Should we do this in the requirements.txt file instead, where we need to describe that these libraries are essential for ART?

Reply:
These are necessary system-level dependencies for `pydub`.